front/src/components/Navigation.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react"
+import {NavLink} from "react-router-dom"
+
+const Navigation = () => {
+  return (
+    <div className="navigation">
+      <ul>
+        <NavLink className={"purple"} to="/">
```

Comment:
J'ai un doute y'a besoin des {} ?
ou on peut mettre <NavLink className="purple" to="/">

Reply:
Effectivement! Merci